server/tests/utils.py

```diff
@@ -0,0 +1,12 @@
+import psycopg2
+from migrate_db import migrate
+
+def standard_seed_db(postgresql):
```

Comment:
"seed" is standard word for load initial data to database.
I add "standard" prefix, because that we will have different initial data for tests.

"populate_db" and "load_test_data" means the same as "seed_db" and they aren't better.

Reply:
I still don't like it, but let it me.